Cargo.toml

```diff
@@ -0,0 +1,140 @@
+[package]
+name = "py2many-tests"
+version = "0.1.0"
+description = "A package for py2many tests"
+edition = "2018"
+include = ["tests/ext_expected/*.rs"]
+
+[dependencies]
+flagset = "*"
+futures = "*"
+pyo3 = "*"
+structopt = "*"
+strum = "*"
+strum_macros = "*"
+lazy_static = "*"
+
+[lib]
```

Comment:
Any way to generate this dynamically, so when new tests are added, entries appear here magically?

Reply:
Ya, that is going to need to be done for https://github.com/adsharma/py2many/issues/158